README.md

```diff
@@ -1,2 +1,21 @@
+Humanoid League Messages
+========================
+
+Standard messages:
+------------------
+
+Standard messages are used for:
+
+- ball_relative (PoseWithCovariance)
+- balls_relative (PoseWithCovarianceArray)
+- goalpost_relative (PoseWithCovariance)
+- goalposts_relative (PoseWithCovarianceArray)
```

Comment:
This is not a standard message, since it is a self defined message.

Reply:
I will call them generalized messages.